portal/src/graphql/portal/OnboardingConfigAppScreen.tsx

```diff
@@ -254,6 +321,7 @@ const OnboardingConfigAppScreenForm: React.FC<OnboardingConfigAppScreenFormProps
         <FormattedMessage id="Onboarding.desc" />
       </Text>
       <IdentitiesListContent form={form} />
+      <PrimaryAuthenticatorsContent form={form} />
```

Comment:
1. Change primary authenticator to OTP
2. Add username
3. See OTP is disabled and STILL selected.